rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -379,4 +386,43 @@ public ResponseEntity<?> scheduleSourceUploadForReleaseComponents()throws TExcep
         HttpStatus status = HttpStatus.ACCEPTED;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @PostMapping(SCHEDULE_URL + "/status")
+    public ResponseEntity<Map<String, Object>> checkServiceStatus(@RequestBody Map<String, String> requestBody) {
+        try {
+            User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+
+            String serviceName = requestBody.get("serviceName");
+            if (serviceName == null || serviceName.isBlank()) {
+                return ResponseEntity.badRequest()
+                        .body(Map.of("error", "Service name is required"));
+            }
+
+            RequestStatus requestStatus = scheduleService.isServiceScheduled(serviceName, sw360User);
+
+            Map<String, Object> response = Map.of(
+                    "serviceName", serviceName,
+                    "isScheduled", requestStatus == RequestStatus.SUCCESS
+            );
+
+            return ResponseEntity.ok(response);
+        } catch (TException e) {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
+                    .body(Map.of("error", "Failed to check service status"));
+        }
+    }
+
+    @GetMapping(SCHEDULE_URL + "/isAnyServiceScheduled")
```

Comment:
```suggestion
    @RequestMapping(value = SCHEDULE_URL + "/isAnyServiceScheduled", method = RequestMethod.GET)
```

Also missing `@Operation` documentation.

Reply:
Suggested changes are not done.

Reply:
In ScheduleAdminController.java class we had following this pattern for all other previous functions as well
** @GetMapping(SCHEDULE_URL + "/isAnyServiceScheduled")** instead of@RequestMapping(value = SCHEDULE_URL + "/isAnyServiceScheduled", method = RequestMethod.GET)

Reply:
Let us follow this for new endpoints and unify later :-)

Reply:
I have updated the code. Please review